inc/functions.php

```diff
@@ -6765,19 +6766,56 @@ function fetch_remote_file($url, $post_data=array())
 
 	if(function_exists("curl_init"))
 	{
+		$can_followlocation = @ini_get('open_basedir') == null && (!@ini_get('safe_mode') || @ini_get('safe_mode') == 'Off');
```

Comment:
@Devilshakerz This check should probably use $this->safemode. I'm also not entirely sure the open_basedir check works under all circumstances. If ini_get is disabled it always returns null regardless of the actual value.